README.md

```diff
@@ -50,9 +50,12 @@ Coming Soon!
 
 Connector runtimes provided by the Community:
 
-| Connector  | JAR with dependencies | Code repository | Documentation | License | Lifecycle |
-| ---------- | --------------------- | --------------- | ------------- | ------- | --------- |
-|            |                       |                 |               |         |           |
+| Connector                 | JAR with dependencies                                                                 | Code repository                                                                   | Documentation                                                                                         | License | Lifecycle |
+|---------------------------|---------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------| ------- | --------- |
+| Office (Word, OpenOffice) | [Community] (https://github.com/camunda-community-hub/zeebe-cherry-officepdf-workers) | [github](https://github.com/camunda-community-hub/zeebe-cherry-officepdf-workers) | [ReadMe](https://github.com/camunda-community-hub/zeebe-cherry-officepdf-workers/blob/main/README.md) |     [Apache 2.0]     |    [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-)|
+| PDF                       | [Community] (https://github.com/camunda-community-hub/zeebe-cherry-officepdf-workers) | [github](https://github.com/camunda-community-hub/zeebe-cherry-officepdf-workers) | [ReadMe](https://github.com/camunda-community-hub/zeebe-cherry-officepdf-workers/blob/main/README.md) |     [Apache 2.0]     |    [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-)|
+| BPMN Operation (message) | [Community] (https://github.com/camunda-community-hub/zeebe-cherry-bpmn-operations) | [github](https://github.com/camunda-community-hub/zeebe-cherry-officepdf-workers) | [ReadMe](https://github.com/camunda-community-hub/zeebe-cherry-bpmn-operations/blob/main/README.md) |     [Apache 2.0]     |    [![](https://img.shields.io/badge/Lifecycle-Stable-brightgreen)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#stable-)|
```

Comment:
This Connector Repo misses an element template. Am I missing something?

Reply:
The connector is integrated into the Cherry Framework. The framework generates automatically the element-template. But I add it for you in a element-templates folder.